### DISCUSSION TITLE
Paid consultation about core pixi.js concepts & optimizing a project.

### DISCUSSION THREAD

**Discussion by yordan-kanchelov** [pixijs/pixijs#8159] [Jobs]
Hey guys, 

Is someone interested in providing me with a consultation about optimizing one pixi.js scene? I feel like creating a boilerplate to showcase our current scene structure and discuss the ways it could be optimized because currently, it's quite heavy in my opinion.  

I feel like from this we can create a showcase and more advanced pixi.js guide because currently, I haven't found such a tutorial except for some forum threads discussing different aspects. 

@ivanpopelyshev @bigtimebuddy if one of you is interested in this or has someone to recommend me who has interest I would be thankful. 

Cheers!

**Comment by bigtimebuddy** [comment]
I'd recommend moving this to the Jobs section on the discussion board. I cannot consult right now, but Ivan may?

**Reply by ivanpopelyshev** [reply]
Nope, I'm too busy :(

**Reply by yordan-kanchelov** [reply]
I moved it :). 

I feel like it would be no more than a full workday and I am not In a hurry for something like that, so I would be glad if at some point we could make it work
